en/power_module/ark_12s_pab_power_module.md

```diff
@@ -12,9 +12,9 @@ Order this module from:
 
 ## Hardware Specifications
 
-- **TI INA226 Digital Power Monitor**
+- **TI INA238 Digital Power Monitor**
 
-  - 0.1mOhm Shunt
+  - 0.0001 Ohm Shunt
```

Comment:
Just FMI, why is it better to present as 0,0001 ohm than 0.1 mOhm?

Reply:
You have to set the param in Ohm, so we just want to minimize the error from the user side, so they can just copy paste it!